src/account/Settings.tsx

```diff
@@ -100,6 +101,7 @@ interface StateProps {
   supportedBiometryType: BIOMETRY_TYPE | null
   shouldShowRecoveryPhraseInSettings: boolean
   hapticFeedbackEnabled: boolean
+  odisV1EOL: boolean
```

Comment:
nit: seems simpler, no?
```suggestion
  odisV1Enabled: boolean
```

Reply:
lol, yes this is better. i updated

Reply:
lol, i'm also fine with that. updated